Source/cmake/WebKitFeatures.cmake

```diff
@@ -267,6 +267,7 @@ macro(WEBKIT_OPTION_BEGIN)
     WEBKIT_OPTION_DEFINE(USE_SKIA "Whether to use Skia instead of Cairo." PRIVATE OFF)
     WEBKIT_OPTION_DEFINE(USE_SYSPROF_CAPTURE "Whether to use libsysprof-capture for tracing." PRIVATE OFF)
     WEBKIT_OPTION_DEFINE(USE_SYSTEM_MALLOC "Toggle system allocator instead of WebKit's custom allocator" PRIVATE ${USE_SYSTEM_MALLOC_DEFAULT})
+    WEBKIT_OPTION_DEFINE(USE_SYSTEM_SYSPROF_CAPTURE "Whether to use a system-provided libsysprof-capture" PRIVATE OFF)
```

Comment:
> Oops, both this and the preexisting `USE_SYSPROF_CAPTURE` option should go in OptionsGTK.cmake and OptionsWPE.cmake instead, since they're not relevant to all platforms. Same for `USE_SKIA`. The idea is that options listed here should be relevant to more than just WPE/GTK.

I'll move the Sysprof options to `Options{WPE,GTK}.cmake`.

OTOH, `USE_SKIA` is *already* cross-port: PlayStation has the plumbing to use it, and WinCairo is getting there as well. Skia doesn't depend on GLib either.

> One more thing: this should be enabled by default, because using the system version is preferable to using the bundled version.

No. I want the bundled one by default, otherwise packagers are more likely to disable the support completely via `USE_SYSPROF_CAPTURE=OFF` instead of setting `USE_SYSTEM_SYSPROF_CAPTURE=OFF`. Trust me, I have packaged software for years (and still do), and I know well how they think. As long as we provide an option to let them pick the system package easily, they won't be angry at us, either.

Reply:
The end result will be distros using the bundled version, since that is default. If this is what you want and the system option is only intended to be an alternative, then what you have now is ideal. But if you want distros to actually use the system sysprof, then system sysprof should be default.

Our standard pattern would work fine here:

```
if (NOT SysProfCapture_FOUND)
    message(FATAL_ERROR "sysprof-capture is required for USE_SYSTEM_SYSPROF_CAPTURE")
endif ()
```

Packagers won't even see `USE_SYSPROF_CAPTURE` unless they go looking for it. They _will_ see `USE_SYSTEM_SYSPROF_CAPTURE` because it'll be at the bottom of the failed build log. So the path of least resistance will not lead toward disabled sysprof-capture.

The message could be reworded for further clarity; currently all our error messages assume some level of familiarity with CMake options. E.g. "sysprof-capture dependency is required because USE_SYSTEM_SYSPROF_CAPTURE is enabled; you may disable with -DUSE_SYSTEM_SYSPROF=OFF"

> OTOH, USE_SKIA is already cross-port: PlayStation has the plumbing to use it, and WinCairo is getting there as well. Skia doesn't depend on GLib either.

Fair point.

Reply:
@mcatanzaro Updated, I still needed to keep usage of the bundled copy while we do not have `libsysprof-capture-4` included in the SDKs. I have filed [bug #277627](https://bugs.webkit.org/show_bug.cgi?id=277627) to track this.